src/Components/Facility/AddInventoryForm.tsx

```diff
@@ -207,9 +208,11 @@ export const AddInventoryForm = (props: any) => {
       setIsLoading(false);
     }
   };
+
   const handleChange = (e: any) => {
     const form = { ...state.form };
-    form[e.target.name] = e.target.value;
+    form[e.name] = e.value;
+    setOptionLabelId(form);
```

Comment:
Am I not understanding `setOptionLabelId`? This doesn't seems to make sense. Could you explain?

Reply:
to add optionvalue in selectformfield component, i am stuck in this optionvalue whole day then I create a state and do this and then it working fine